typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -97,6 +98,10 @@ export interface WellsLayerProps extends ExtendedLayerProps {
      * For example depth of z = 1000 corresponds to -1000 on the z axis. Default true.
      */
     ZIncreasingDownwards: boolean;
+    optimizedRendering: boolean;
```

Comment:
Can you add some doc ?

Reply:
optimizedInteraction is documented, but not optimizedRendering

Reply:
Doc is missing for `optimizedRendering`

Reply:
That one is for internal use (not to be set by user) but I put some documentation there.